core/src/services/iframe.js

```diff
@@ -113,6 +113,18 @@ class IframeClass {
   }
 
   setOkResponseHandler(config, component, node) {
+    const viewGroup = config.iframe?.vg;
+    const viewGroupSettings = config.iframe?.contentWindow?.parent['Luigi']?.config?.navigation?.viewGroupSettings;
+    const noClientCheck = viewGroupSettings && viewGroup ? viewGroupSettings[viewGroup]?.noClientCheck : false;
+
+    /**
+     * check for `noClientCheck` attribute
+     * when set to `true`, it prevents a navigation check when reactivating the iframe
+     */
+    if (noClientCheck) {
+      return;
+    }
+
```

Comment:
Please never do sth. like iframe..contentWindow.xyz as it will fail for cross-origin cases
```suggestion
    if (config?.iframe?.vg && NavigationHelpers.getViewGroupSettings(config.iframe.vg)?.noClientCheck) {
      return;
    }
```

Reply:
same check must be done in `initHandshakeFailed` after first config.iframe.luigi check